Library/Homebrew/cask/auditor.rb

```diff
@@ -20,9 +28,7 @@ def audit_download?
       @audit_download
     end
 
-    def quarantine?
-      @quarantine
-    end
+    attr_predicate :audit_appcast?, :quarantine?
```

Comment:
Move this below `attr_reader`.

Reply:
not sure if there should be a free line between attr_reader and attr_predicate, i found both styles being in use